Tensile/Source/lib/source/Debug.cpp

```diff
@@ -95,6 +95,11 @@ namespace Tensile
         return m_value & 0x2000;
     }
 
+    bool Debug::printWinningKernelName() const
+    {
+        return m_value & 0x4000;
```

Comment:
does the debug bit for `printWinningKernelName()` intentionally collide with `printLookupEfficiency()`? They both AND on the same mask 0x4000

Reply:
Thanks for catching that, the rest were set up in increasing order and I missed that one. I'll change it to 0x6000 and maybe move printLookupEfficiency just before it to keep the ordering

Reply:
That last comment should have said 0x8000, I've applied that change in the newest commit